README.md

```diff
@@ -229,19 +349,19 @@ To illustrate the capabilities of the COS API, consider the following example wh
 On Site A, a web application stores a large language model in COS.
 
 ```js
-// The hash of the file we want to access.
-const hash = {
+// The hash of the desired file.
+const hashes = [{
```

Comment:
For legibility, we could…
* use only the hash object here `const hash = {}`
* invoke `requestFileHandles([hash])`
* and later compare `controlHash !== hash.value`

Reply:
My thinking was that it's easy to overlook the square brackets in `requestFileHandles([hash])`, so I went for consistency. I'll change it for improved legibility.